client/my-sites/navigation/index.jsx

```diff
@@ -92,9 +105,15 @@ export default withCurrentRoute(
 	connect( ( state, { currentSection } ) => {
 		const sectionGroup = currentSection?.group ?? null;
 		const siteId = getSelectedSiteId( state );
-		const { shouldShowGlobalSidebar } = useGlobalSidebar( siteId, sectionGroup );
+		const adminInterface = getSiteOption( state, siteId, 'wpcom_admin_interface' );
+		const { shouldShowGlobalSidebar, shouldShowGlobalSiteSidebar } = useGlobalSidebar(
+			siteId,
+			sectionGroup
+		);
 		return {
 			isGlobalSidebarVisible: shouldShowGlobalSidebar,
+			// Global Site View should be limited to classic interface users only for now.
+			isGlobalSiteSidebarVisible: shouldShowGlobalSiteSidebar && adminInterface === 'wp-admin',
```

Comment:
I think we might want the adminInterface check in the hook so shouldShowGlobalSiteSidebar can be checked elsewhere. I'll create a follow up.

Reply:
https://github.com/Automattic/dotcom-forge/issues/5609

Reply:
Note same check down here https://github.com/Automattic/wp-calypso/pull/87477/files?w=1#diff-41e334c729a3d39543a579d460a44cd819298d62cb76a12c0b9e2f8087cdff5eR118